frontend-api/src/test/java/uk/gov/di/authentication/frontendapi/lambda/CheckUserExistsHandlerTest.java

```diff
@@ -219,6 +224,39 @@ void shouldReturn400IfEmailAddressIsInvalid() {
                         PersistentIdHelper.PERSISTENT_ID_UNKNOWN_VALUE);
     }
 
+    @Test
+    void shouldReturn400IfUserAccountIsLocked() throws Json.JsonException {
+        String email = "joe.bloggs@digital.cabinet-office.gov.uk";
+        usingValidSession();
+        when(authenticationService.userExists(eq(email))).thenReturn(false);
+        when(codeStorageService.getIncorrectPasswordCount(email)).thenReturn(5);
+
+        APIGatewayProxyRequestEvent event = new APIGatewayProxyRequestEvent();
+        event.setBody("{ \"email\": \"joe.bloggs@digital.cabinet-office.gov.uk\" }");
+        event.setHeaders(
+                Map.of(
+                        "Session-Id",
+                        session.getSessionId(),
+                        CLIENT_SESSION_ID_HEADER,
+                        CLIENT_SESSION_ID));
+        event.setRequestContext(contextWithSourceIp("123.123.123.123"));
+        APIGatewayProxyResponseEvent result = handler.handleRequest(event, context);
+
+        assertEquals(400, result.getStatusCode());
+
```

Comment:
Could try something like and also add an assert to check the expected error response
```suggestion
         assertThat(result, hasStatus(400));
        assertThat(result, hasJsonBody(ErrorResponse.ERROR_1045));
```